tests/Entity/DocumentDetailsTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace YotiTest\Entity;
+
+use Yoti\Entity\DocumentDetails;
+use YotiTest\TestCase;
+
+class DocumentDetailsTest extends TestCase
+{
+    /**
+     * @var DocumentDetails
+     */
+    public $dummyDocumentDetails;
+
+    public function setup()
+    {
+        $dummyValue = 'PASSPORT GBR 01234567 2020-01-01';
+        $this->dummyDocumentDetails = new DocumentDetails($dummyValue);
+    }
+
+    public function testShouldParseOneOptionalAttribute()
+    {
+        $document = new DocumentDetails('PASSPORT GBR 01234567 2020-01-01');
+        $this->assertEquals('PASSPORT', $document->getType());
+        $this->assertEquals('GBR', $document->getIssuingCountry());
+        $this->assertEquals('01234567', $document->getDocumentNumber());
+        $this->assertEquals('2020-01-01', $document->getExpirationDate()->format('Y-m-d'));
+    }
+
+    public function testShouldParseTwoOptionalAttributes()
+    {
+        $document = new DocumentDetails('DRIVING_LICENCE GBR 1234abc 2016-05-01 DVLA');
+        $this->assertEquals('DRIVING_LICENCE', $document->getType());
+        $this->assertEquals('GBR', $document->getIssuingCountry());
+        $this->assertEquals('1234abc', $document->getDocumentNumber());
+        $this->assertEquals('2016-05-01', $document->getExpirationDate()->format('Y-m-d'));
+        $this->assertEquals('DVLA', $document->getIssuingAuthority());
+    }
+
```

Comment:
I know it's not in the Java SDK yet, but I wonder if we should add another test...
_shouldIgnoreAThridOptionalAttribute_
...and pass it a String like...
_'DRIVING_LICENCE GBR 1234abc 2016-05-01 DVLA someThirdAttribute'_

Reply:
I think this is going against the format we are expecting from the Document Details value.
As we are expecting 5 words not 6.

Reply:
True, but it doesn't hurt to prove that we won't fail if a 6th attribute starts appearing on DocumentDetails.  We would just ignore the 6th attribute, and process the first 3-5 as normal.

Reply:
Ok - makes sense.